### ISSUE TITLE
[BUG] Heading tags should be in order

### ISSUE THREAD

**Issue by justyna-13** [vuestorefront/storefront-ui#2010]
**Describe the bug**
Heading tags should be in order. There are pages with unordered headings tags for example Product Page

**Comment by iqbaltawakkal** [comment]
hi @justyna-13 I want to create PR for this issue but did check product page still not sure what you're expecting, hint/guide will be appreciated, thanks in advance.

**Comment by iqbaltawakkal** [comment]
@justyna-13 got it, but should we adjust font size after re-ordering the headings? because default font-size for each (higher heading) level feel too big.

**Comment by justyna-13** [comment]
@iqbaltawakkal  Yes, everything should look like before changes so we need these adjustments

**Comment by justyna-13** [comment]
@iqbaltawakka  what is the progress of this issue? :slightly_smiling_face:
